File: GO/ex1009.go
package main

import(
	"fmt"
)

func after_bunus(salary float64, bonus float64) float64{
	return salary + bonus * 0.15
}

func main(){
	var name string
	var salary, bonus float64
	fmt.Scanf("%s", &name)
	fmt.Scanf("%f", &salary)
	fmt.Scanf("%f", &bonus)
	fmt.Printf("TOTAL = R$ %.2f\n", after_bunus(salary, bonus))
}

File: C/ex1009.c
#include <stdio.h>
int main(){
    char nome[256];
    double salary, bonus;
    scanf("%s", nome);
    scanf("%lf %lf",&salary, &bonus);
    salary+=bonus*0.15;
    printf("TOTAL = R$ %.2lf\n", salary);
}
